src/app/main.cpp

```diff
@@ -255,16 +255,27 @@ int main(int argc, char *argv[])
     }
 }
 
+#if !defined Q_OS_WIN && !defined Q_OS_HAIKU
+void reportToUser(const char* str)
+{
+    const size_t strLen = strlen(str);
+    if (write(STDERR_FILENO, str, strLen) < static_cast<ssize_t>(strLen)) {
+        auto dummy = write(STDOUT_FILENO, str, strLen);
```

Comment:
Don't know the logic here, if stderr failed then stdout will probably fail as well.